test/visual-diff/visual-tests

```diff
@@ -600,5 +600,14 @@
       ],
       "interactive_tests": "examples/visual-tests/amp-story/amp-story-page-attachment-desktop.js"
     },
+    {
+      "url": "examples/visual-tests/amp-user-notification/amp-user-notification-blocking.amp.html",
+      "name": "amp-user-notification",
+      "viewport": {"width": 400, "height": 600},
+      "loading_complete_css": [
+        "amp-user-notification.amp-active",
+      ],
+      "interactive_tests": "examples/visual-tests/amp-user-notification/dismiss-amp-user-notification.js"
```

Comment:
Rename this file to `amp-user-notification-blocking.js` (or `amp-user-notification.js`, if you accept the previous comment)